apps/api/admin/connect-to-dal.ts

```diff
@@ -0,0 +1,17 @@
+import { DalService } from '@novu/dal';
+
+const dalService = new DalService();
+
+export async function connect(databaseQuery: () => Promise<void>) {
+  try {
+    await dalService.connect(process.env.MONGO_URL);
+    await databaseQuery();
+  } catch (e) {
+    console.error(e);
+  } finally {
+    if (dalService.isConnected()) {
+      await dalService.disconnect();
+    }
+    process.exit(0);
+  }
+}
```

Comment:
The util function allows you to connect to the Mongo through the DalService and run the database queries. In case of any error, it prints it and closes the DB connection.